nemo/collections/multimodal/speech_llm/parts/utils/data_utils.py

```diff
@@ -155,3 +155,15 @@ def align_feat_seq_list(
         new_seq_list.append(new_seq)
         new_seq_len_list.append(new_seq_len)
     return new_seq_list, new_seq_len_list
+
+
+def to_cuda(inputs, non_blocking=True):
```

Comment:
remove this function, use this one https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/mixins/transcription.py#L70

Reply:
Done